apps/payment/views.py

```diff
@@ -129,6 +129,11 @@ def webshop_pay(request):
             try:
                 stripe.api_key = settings.STRIPE_PRIVATE_KEYS["prokom"]
 
+                if not order_line.is_valid():
+                    error = "Ordren er ikke gyldig."
+                    messages.error(request, error)
+                    return HttpResponse(error, content_type="text/plain", status=500)
```

Comment:
HTTP 500 for an invalid order is a bit much, eh? The standard HTTP 400 BAD REQUEST should be fine. (An invalid order would be a client mistake, not a server side one, right?)